libs/features/mas/src/checkout-mixin.js

```diff
@@ -100,7 +100,6 @@ export function CheckoutMixin(Base) {
         }
 
         async render(overrides = {}) {
-            if (!this.isConnected) return false;
```

Comment:
is it to fix some issue? do you have more details?

Reply:
@3ch023 I asked them.  Since `sp-button` doesn't have checkout-link capabilities, instead we will render a checkout-link but not in the DOM and just forward clicks to it. 
But the DOM element must be rendered even if it is not connected to DOM which is fine IMO.

Reply:
i am good if there is no impact on events we throw for CCD